yarn.lock

```diff
@@ -17574,6 +17574,22 @@ tryer@^1.0.1:
   resolved "https://registry.yarnpkg.com/tryer/-/tryer-1.0.1.tgz#f2c85406800b9b0f74c9f7465b81eaad241252f8"
   integrity sha512-c3zayb8/kWWpycWYg87P71E1S1ZL6b6IJxfb5fvsUgsf0S2MVGaDhDXXjDMpdCpfWXqptc+4mXwmiy1ypXqRAA==
 
+ts-jest@26.5.3:
```

Comment:
why we updated yarn lock file?

Reply:
was failing on format otherwise
```
+ git status --untracked-files=no --porcelain

+ [ -z  M yarn.lock ]

+ echo Running format is required

Running format is required

+ exit 1
```

Reply:
```
"ts-jest": "26.5.3",
"ts-node": "9.1.1",
```
These values are coming from commander, as specified in the issue `using lisk generate:plugin command`

Reply:
Ah I see, need to update commander